core/services/relay/evm/write_target.go

```diff
@@ -1,21 +1,39 @@
+//nolint:gosec // disable G115
 package evm
 
 import (
+	"bytes"
 	"context"
+	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
```

Comment:
Can we stick to std lib?
```suggestion
	"errors"
```

Reply:
fixed